mesonbuild/interpreter/interpreter.py

```diff
@@ -1066,8 +1066,11 @@ def _do_subproject_cargo(self, subp_name: str, subdir: str,
                 [os.path.join(subdir, 'Cargo.toml')])
 
     @typed_pos_args('get_option', str)
-    @noKwargs
-    def func_get_option(self, nodes: mparser.BaseNode, args: T.Tuple[str],
+    @typed_kwargs(
+        'get_option',
+        KwargInfo('fallback', str, default='', since='1.8.0'),
```

Comment:
Usually in this case we end up just using `object`, which allows anything through. I think what you really want is:
```python
KwargInfo('fallback', (NoneType, object), since='1.8.0')
```
object will catch any valid Meson types, while the explicit NoneType tells the read this is optional.